cypress/integration/plugins/gantt-chart-dashboards/gantt_ui.spec.js

```diff
@@ -5,8 +5,12 @@
 
 /// <reference types="cypress" />
 
+import customParseFormat from 'dayjs/plugin/customParseFormat';
+import dayjs from 'dayjs';
```

Comment:
I don't see this installed is it included in another dependency?

Reply:
True, We'll update the PR to use moment as mentioned here: https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/515. Thanks for the catch.

Reply:
Has been updated

Reply:
@kavilla we updated both the PRs to use `moment js`